PowerShell/ScubaGear/Testing/Unit/PowerShell/Orchestrator/Invoke-ScubaConfig.Tests.ps1

```diff
@@ -0,0 +1,97 @@
+$OrchestratorPath = '../../../../Modules/Orchestrator.psm1'
+$ScubaConfigPath = '../../../../Modules/ScubaConfig/ScubaConfig.psm1'
+$ConnectionPath = '../../../../Modules/Connection/Connection.psm1'
+
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $OrchestratorPath) -Function 'Invoke-SCuBA' -Force
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $ScubaConfigPath)
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $ConnectionPath) -Function Disconnect-SCuBATenant
+
+InModuleScope Orchestrator {
+    Describe -Tag 'Orchestrator' -Name 'Invoke-Scuba with Config' {
+        BeforeAll {
+            function Remove-Resources {}
```

Comment:
This is the third time I am raising the issue of redefining the function before mocking it.  Is it your intent not to make the change? Am I reviewing the correct code?

Reply:
Sorry I did implement the change but somehow the modified file was missing from the commit.  It has now been pushed.  Feel bad that this happened.

Reply:
One of these functions is restored as per following commit

Add local funcion declaration for Disconnect-SCuBATenant in Orchestrator ScubaConfig tests
    This will enable this external module to run with top level Pester-Test
    Reference:  https://www.logitblog.com/mastering-mocking-with-pester-commandnotfoundexception/

I suspect this is why the other function declaration  were present ( but no documentation about why they were thee )